README.md

```diff
@@ -9,6 +9,21 @@ Sandbox UI for the smartevents application.
 - Node 16
 - npm 7
 
+### Code formatting
+
+The project requires the code to be formatted by `prettier`. This is done automatically as a `pre-commit` hook. To activate the hook, it is needed to run the commands below after you cloned the project.
+
+```
+npm install
+npm run prepare
+```
```

Comment:
The `prepare` npm script [runs automatically](https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-scripts) on `npm install`. It's not necessary to run it manually.

Reply:
@kelvah I have updated the readme